### PR TITLE
Added recipe for routes

### PR DESCRIPTION
**Pull request by pmlandwehr**
Adding a recipe for `routes`, since it's required for the extras in the latest `cherrypy`. (ref. https://github.com/conda-forge/cherrypy-feedstock/pull/8)

### CODE REVIEW

File: recipes/routes/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - repoze.lru >=0.3
+
+test:
+  imports:
+    - routes
+
+about:
+  home: https://routes.readthedocs.io/
+  license: MIT
+  license_family: MIT
```
Comment (synapticarbors) [inline recipes/routes/meta.yaml]: please add

`license_file: LICENSE.txt`

**Comment by pmlandwehr** [comment]
Not sure what's up here - for some reason invoking curl is causing AppVeyor to crash.

**Comment by pmlandwehr** [comment]
This is the issue: https://github.com/conda-forge/conda-smithy/issues/487

**Comment by pmlandwehr** [comment]
(Open/close to redo the CIs with the latest smithy.)

**Comment by jakirkham** [comment]
Thanks @pmlandwehr.

**Pull request merged**